public/javascripts/jquery/jquery-1.7.1.min.js

```diff
@@ -0,0 +1,4 @@
+/*! jQuery v1.7.1 jquery.com | jquery.org/license */
```

Comment:
I'm confused why this branch would have a file called `jquery-1.7.1.min.js` in it. Aren't we trying to get _rid_ of jQuery 1.x?